### ISSUE TITLE
Sidebar dropdown not style on widgets

### ISSUE THREAD

**Issue by rubengc** [danieltj27/Dark-Mode#63]
On widgets screen, clicking on a widget shows a dropdown where you are able to select the sidebar to place it, and this options are not styled:
![widgets-issue](https://user-images.githubusercontent.com/5462480/34454796-23d409e6-ed73-11e7-97d0-6f73fa7247ab.PNG)

It requires the next rules:
```
.widgets-chooser li {
    background-color: #50626f;
    border-color: #191f25;
}

.widgets-chooser li:hover, .widgets-chooser li:focus {
    background-color: #37444c;
}
```

Edit: Change background color to same applied to default buttons

**Comment by danieltj27** [comment]
Thanks for reporting this, looks like something got missed somewhere.

**Comment by danieltj27** [comment]
Fixed in commit bf49c03. Thanks!
